packages/components/src/DateTimePickers/views/ViewLayout/ViewLayout.scss

```diff
@@ -6,16 +6,18 @@
 
 	.header {
 		display: flex;
-		height: 5rem;
+		height: 2.5rem;
```

Comment:
You should not set a height. 
In the header, define the paddings only (as described in the guideline). The height will be set automatically depending on the paddings and the buttons height.

Reply:
updated, Thanks